src/backend/distributed/sql/udfs/citus_stat_activity/latest.sql

```diff
@@ -0,0 +1,23 @@
+DROP VIEW IF EXISTS pg_catalog.citus_stat_activity;
+DROP TYPE IF EXISTS pg_catalog.global_pid_and_pg_stat_activity;
+
+CREATE TYPE citus.global_pid_and_pg_stat_activity AS (global_pid bigint, pg_stat_activity pg_stat_activity);
```

Comment:
What happens to `global_pid_and_pg_stat_activity` whe `pg_stat_activity` changes in the next PG version?

Reply:
We drop it and re-create it

Reply:
I meant, PG version change. We execute these scripts on Citus version upgrades.

Reply:
I added a `CREATE VIEW` query to the PG upgrade function. So now it updates the view.

Reply:
Cool, maybe quickly check with Nils if this sounds OK for him as well?